app/components/UtilizationDatum.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+
+export const UtilizationDatum = ({
+  name,
+  amount,
+}: {
+  name: 'Provisioned' | 'Quota' | 'Capacity'
```

Comment:
```suggestion
  name: 'Provisioned' | 'Allocated' | 'Capacity'
```
Probably best that we stay consistent with the API